packages/zksend/src/experimental/wallet/index.ts

```diff
@@ -313,12 +321,14 @@ export class StashedWallet implements Wallet {
 	};
 }
 
-export function registerStashedWallet(
+export async function registerStashedWallet(
 	name: string,
 	{
 		origin,
+		useDynamicMetadata = true,
```

Comment:
IMO for the initial release we should not let folks opt-out of this, since we know we'll be doing a day one update.